src/jit/x64/emit.dasc

```diff
@@ -1650,6 +1794,24 @@ void MVM_jit_emit_primitive(MVMThreadContext *tc, MVMJitCompiler *compiler, MVMJ
         |2:
         break;
     }
+    case MVM_OP_isinvokable: {
+        MVMint16 dst = ins->operands[0].reg.orig;
+        MVMint16 src = ins->operands[1].reg.orig;
+        | mov TMP1, aword WORK[src];
+        | mov TMP1, OBJECT:TMP1->st;
+        | mov TMP2d, dword STABLE:TMP1->invoke;
+        | cmp TMP2d, dword MVM_6model_invoke_default;
+        | jne >1;
+        | mov TMP3, STABLE:TMP1->invocation_spec;
+        | cmp TMP3, 0;
+        | jne >1;
+        | mov aword WORK[dst], 0;
+        | jmp >2;
+        |1:
+        | mov aword WORK[dst], 1;
+        |2:
```

Comment:
Minor potential improvement; most jumps here can be eliminated by setting up a 1 (in TMP2 or something) and using AND with the interesting values.